### ISSUE TITLE
bug -- formula rebuild 

### ISSUE THREAD

**Issue by thomasgillis** [thomasgillis/dbs#3]
when running independently

```bash
make fftw
```

and then

```bash
make p4est
```

`fftw` is rebuild even though the file `fftw.complete` is up to date in the `$(PREFIX)` dir

**Comment by thomasgillis** [comment]
As reported by @pbalty  the issue comes from the dependency on `$(COMP_DIR)` in the `.complete` formula.
This has been solved by b9d492535a78499e2b1830dd5f356bb0b0b16de0

**Comment by thomasgillis** [comment]
we still have issues with the `ompi` formula:

```bash
[euthomasg@vglogin0002 dbs]$ CLUSTER=empty make clean

[euthomasg@vglogin0002 dbs]$ CLUSTER=empty make ompi --trace
ucx.mak:35: target '/ceph/hpc/home/euthomasg/test_dbs/ucx.complete' does not exist
touch /ceph/hpc/home/euthomasg/test_dbs/ucx.complete
ofi.mak:37: target '/ceph/hpc/home/euthomasg/test_dbs/ofi.complete' does not exist
touch /ceph/hpc/home/euthomasg/test_dbs/ofi.complete
ompi.mak:51: update target '/ceph/hpc/home/euthomasg/test_dbs/ompi.complete' due to: ucx ofi
touch /ceph/hpc/home/euthomasg/test_dbs/ompi.complete

[euthomasg@vglogin0002 dbs]$ CLUSTER=empty make ompi --trace
ompi.mak:51: update target '/ceph/hpc/home/euthomasg/test_dbs/ompi.complete' due to: ucx ofi
touch /ceph/hpc/home/euthomasg/test_dbs/ompi.complete
```
